core/common/src/main/java/alluxio/underfs/UnderFileSystemConfiguration.java

```diff
@@ -23,14 +23,21 @@
  * {@link RuntimeException} if the key is not found in both configurations..
  */
 public final class UnderFileSystemConfiguration {
+  private final boolean mReadOnly;
+  private final boolean mShared;
```

Comment:
I would prefer to add `MountInfo` in that case, to avoid needing to introduce additional fields and methods if more options are added